x-pack/test/apm_api_integration/basic/tests/services/index.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { FtrProviderContext } from '../../../common/ftr_provider_context';
+
+export default function servicesIntegrationTests({ loadTestFile }: FtrProviderContext) {
+  describe('Services', function () {
+    loadTestFile(require.resolve('./annotations'));
+    loadTestFile(require.resolve('./top_services'));
+    loadTestFile(require.resolve('./agent_name'));
+    loadTestFile(require.resolve('./transaction_types'));
+  });
```

Comment:
I think the re-org is great but it would be easier to read if all the `require.resolve` was in a single file. Still a good idea to group in `describe` blocks like here.

Reply:
https://github.com/elastic/kibana/pull/72257/commits/753b762bacba51eb41e564896aee09b07c2e998f